editor/block-mover/mover-label.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * @typedef blockMoverLabelOptions
+ * @property {string}  type       Block type - in the case of a single block, should
+ *                                define its 'type'. I.e. 'Text', 'Heading', 'Image' etc.
+ * @property {number}  firstIndex The index (position - 1) of the first block selected.
+ * @property {boolean} isFirst    This is the first block.
+ * @property {boolean} isLast     This is the last block.
+ * @property {number}  dir        Direction of movement (> 0 is considered to be going
+ *                                down, < 0 is up).
+ */
+
+/**
+ * @typedef multiBlockMoverLabelOptions
+ * @property {number}  firstIndex The index (position - 1) of the first block selected.
+ * @property {boolean} isFirst    This is the first block.
+ * @property {boolean} isLast     This is the last block.
+ * @property {number}  dir        Direction of movement (> 0 is considered to be going
+ *                                down, < 0 is up).
+ */
+
+/**
+ * Wordpress dependencies
+ */
+import { __, sprintf } from 'i18n';
+
+/**
+ * Return a label for the block movement controls depending on block position.
+ *
+ * @param  {number}                 selectedCount Number of blocks selected.
+ * @param  {blockMoverLabelOptions} options       Object options.
+ * @return {string}                               Label for the block movement controls.
+ */
+export function getBlockMoverLabel( selectedCount, { type, firstIndex, isFirst, isLast, dir } ) {
```

Comment:
I'm not sure I understand the distinction between `selectedCount` as a standalone argument vs. `blockMoverLabelOptions` object. Why not have these all as named arguments `( selectedCount, type, firstIndex, isFirst, isLast, dir )` or all as object arguments `( { selectedCount, type, firstIndex, isLast, dir } )`?